llvm/include/llvm/ADT/GenericUniformityImpl.h

```diff
@@ -427,6 +432,7 @@ template <typename ContextT> class GenericUniformityAnalysisImpl {
 
   // Recognized cycles with divergent exits.
   SmallPtrSet<const CycleT *, 16> DivergentExitCycles;
+  SmallVector<UseOutsideCycleInfoT, 4> UsesOutsideCycle;
```

Comment:
Is it correct that this only tracks uses outside a cycle with divergent exits? This should be in the name or at least in a comment.